Header header
string gui_name
string cam_view_mode
string cam_motion_mode
string image3D
string image_surr
string cam_op
string record_op
